roles/ckan/templates/kubernetes/ckan.yaml

```diff
@@ -149,8 +149,10 @@ spec:
               value: "{{ ckan_postgres_password }}"
             - name: PIPENV_DONT_LOAD_ENV
               value: "1"
+{% if fjelltopp_cloud_provider == 'azure' %}
             - name: POSTGRES_HOST
               value: "{{ db_fqdn }}"
+{% endif %}
```

Comment:
```suggestion
```
Advice from @cooper667 is just to delete these lines.  This env var shouldn't be needed.